tofu/mag/test_magFieldLines.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+# vim: tabstop=4 expandtab shiftwidth=4 softtabstop=4
+# Also if needed: retab
```

Comment:
seems like this is specific to your vim config, and not necessary to add ?

Reply:
Agreed, anything specific to a contributor's working environment shall be removed